examples/stocks/lib/stock_settings.dart

```diff
@@ -225,7 +225,7 @@ class StockSettingsState extends State<StockSettings> {
           )
         ),
         new DrawerItem(
-          icon: Icons.gradient,
+          icon: new Icon(Icons.gradient),
```

Comment:
Interesting.

This makes code that uses a raw `Icon` look kinda weird. For example:

```dart
  @override
  Widget buildChild() {
    return new ListItem(
      enabled: config.enabled,
      leading: new FadeTransition(
        opacity: _opacity,
        child: _controller.isDismissed ? null : Icons.done
      ),
      title: config.child
    );
  }
```

It's not obvious that `Icons.done` is a Widget.

I dunno. We should decide one way or the other on this before I land because it affects what code gets broken (either way it's a backwards-incompatible change).